parsers/julia.c

```diff
@@ -1037,14 +1045,14 @@ static void parseFunction (lexerState *lexer, vString *scope, int parent_kind)
             advanceToken(lexer, true);
         }
 
-        addTag(name, NULL, vStringValue(arg_list), K_FUNCTION, line, pos, scope, parent_kind);
+        addTag(name, NULL, vStringValue(arg_list), K_FUNCTION, line, pos, local_scope, local_parent_kind);
         //addToScope(scope, name);
         //parseExpr(lexer, true, K_FUNCTION, scope);
```

Comment:
For now, the test `function_scope.d` can't tag `func5`

If we uncomment these 2 lines, it's expected to tag `func5`, but the result is only `func1` and `func3` are tagged.

My guess is we have to move the lexer to the right place before `parseExpr`.

Reply:
Fixed.